Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.UI.ts

```diff
@@ -1166,19 +1163,13 @@ namespace Remotion.BocAutoCompleteReferenceValue
                 return;
 
             this.element = document.createElement("div");
-            this.element.setAttribute('role', 'listbox');
-            this.element.setAttribute('id', this.options.selectListID);
             this.element.setAttribute('class', this.options.resultsClass);
             this.element.style.position = 'fixed';
             LayoutUtility.Hide(this.element);
 
             this.input.closest('div, td, th, body')!.appendChild(this.element);
 
-            this.options.combobox.setAttribute('aria-owns', this.options.selectListID);
```

Comment:
not sure if removing the `aria-owns` is correct.

Reply:
> This is essentially returning to the 1.0 pattern with one small modification -- use aria-controls to specify the input-popup relationship instead of aria-owns.

See https://github.com/w3c/aria/wiki/Resolving-ARIA-1.1-Combobox-Issues. If I understand correctly, `aria-controls` replaces `aria-owns` completely in ARIA 1.2 implemenation